README.md

```diff
@@ -170,6 +170,22 @@ app.use('/proxy', proxy('www.google.com', {
 }));
 ```
 
+#### bodyPassthrough
+
+The ```bodyPassthrough``` option allows you to ensure the req body is never parsed. This is useful for large
+uploads where it would be inefficient to hold the data
+in memory.
+
+This defaults to false in order to preserve legacy behavior. When true, no action will be taken on the body and accordingly ```req.bodyContent``` will no longer be set.
+
+Note that this overrides ```reqAsBuffer``` and ```reqBodyEncoding``` below.
```

Comment:
Let's add a note to ```reqAsBuffer``` and ```reqBodyEncoding``` as well.   Something like "Ignored if ```parseReqBody``` is set to false."